sdk/core/Azure.Core.Experimental/src/ResponseStatusOption.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure
+{
+    /// <summary>
+    /// ResponseStatusOption controls the behavior of an operation based on the status code of a response.
+    /// </summary>
+    public enum ResponseStatusOption
+    {
+        /// <summary>
+        /// Indicates that an operation should throw an exception when the response indicates a failure.
+        /// </summary>
+        Default = 0,
+        /// <summary>
+        /// Indicates that an operation should not throw an exception when the response indicates a failure.
+        /// </summary>
+        NoThown = 1,
```

Comment:
```suggestion
        NoThrow = 1,
```

Reply:
Actually I realized you can't do just inline fixes since we check in gen code that needs redone. :(